src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -488,6 +488,10 @@ private function registerFragmentsConfiguration(array $config, ContainerBuilder
             return;
         }
 
+        if (!$container->hasParameter('fragment.renderer.hinclude.global_template')) {
```

Comment:
This depends on the order of the calls between `registerFragmentsConfiguration` and `registerTemplatingConfiguration`. Here, we want the configuration on `fragments` to override the one on `templating`, but that's currently the other way around.

I would do it differently. If it's set in both configuration, throw an exception (as it means that the configuration is ambiguous).

Reply:
I understand now, I didn't have that in mind, thanks for the clarification, I'm going to modify in that way. Thanks for the review !